build.gradle

```diff
@@ -66,6 +67,7 @@ dependencies {
     compile "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
     compile "uk.co.jemos.podam:podam:$podamVersion"
     compile "org.projectlombok:lombok:$lombokVersion"
+    compile "io.github.classgraph:classgraph:$classGraphVersion"
```

Comment:
do we neeed this dependency. Pact Gen istelf already has ist mechanism to detect annotations